test/metrics/metrics_test.go

```diff
@@ -23,21 +23,41 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func TestHostOperatorVersionMetrics(t *testing.T) {
+func TestOperatorVersionMetrics(t *testing.T) {
 
-	// given
-	awaitilities := WaitForDeployments(t)
```

Comment:
it's just a detail, but you could have kept it there since you need to wait for the deployments only once.